airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -0,0 +1,231 @@
+/*
```

Comment:
this diff is crap. basically AbstractJdbcSource has all the hard stuff now. JdbcSource imeplements it just like any other jdbc compliant db would.